canonical-web-and-design/ubuntu.com
# Temporaritly remove /gov page

## Done

- Remove /gov pages
- Add redirect to /desktop/organisations
- Removed from the enterprise mega menu (I updated the copy doc as well)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/gov
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you are redirected to http://0.0.0.0:8001/desktop/organisations
